src/openms/source/FORMAT/HANDLERS/MzMLHandler.cpp

```diff
@@ -3538,16 +3543,19 @@ namespace OpenMS
 
     ControlledVocabulary::CVTerm MzMLHandler::getChildWithName_(const String& parent_accession, const String& name) const
     {
-      std::set<String> terms;
-      cv_.getAllChildTerms(terms, parent_accession);
-      for (std::set<String>::const_iterator it = terms.begin(); it != terms.end(); ++it)
+      ControlledVocabulary::CVTerm res = ControlledVocabulary::CVTerm();
```

Comment:
Isn't `ControlledVocabulary::CVTerm res;` enough here?

Reply:
Yep, thanks for pointing it out!